app/jobs/course/discussion/post/codaveri_feedback_rating_job.rb

```diff
@@ -8,7 +8,9 @@ class Course::Discussion::Post::CodaveriFeedbackRatingJob < ApplicationJob
   #
   # @param [Course::Discussion::Post::CodaveriFeedback] codaveri_feedback Feedback with rating to send to Codaveri
   def perform_tracked(codaveri_feedback)
-    Course::Discussion::Post::CodaveriFeedbackRatingService.
-      send_feedback(codaveri_feedback)
+    ActsAsTenant.without_tenant do
```

Comment:
Not really. We want to get the course information, but it would require a tenant. As such, to access the course inside the job, we add it inside without_tenant.